src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -37,6 +37,7 @@ object Backend {
  * Trait to be implemented by concrete backends.
  */
 trait Backend {
+  type BackendCall <: backend.BackendCall
```

Comment:
@mcovarr This makes the abstract type name compliant to the style guide but OTOH we had to acquire the extra cruft on the rhs. The official guide is a fickle beast it would seem.